### PR TITLE
created my personal page

### PR DESCRIPTION
**Pull request by tianhaopx**
I finished step 3 for OLE intern, please review my personal page.

### CODE REVIEW

File: pages/tianhaopx.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+## Tianhao Guo
+
+#### Worcester, MA, EDT
+
+#### macOS Sierra 10.12
+
+I am a master student major in Computer Science at Worcester Polytechnic Institute.
```
Comment (waryhermit) [inline pages/tianhaopx.md]: Could you add a few more lines about yourself?

File: pages/tianhaopx.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+Here's my skills:
```
Comment (waryhermit) [inline pages/tianhaopx.md]: What have you made or worked on with these skills?

File: pages/tianhaopx.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+- JavaScript
+- MySQL
+- MongoDB
+- Hadoop
+- Linux/Unix
+- Git
+
+I'm a team player, and I'm excited to joining you guys!
+
+Connect me on Linkedin: https://www.linkedin.com/in/tianhaoguo
```
Comment (waryhermit) [inline pages/tianhaopx.md]: Could you make `Connect with me on Linkedin` send you to the link instead of pasting it there?

**Comment by tianhaopx** [comment]
Page updated now according to reviews.

**Pull request merged**